config-legacy.example.toml

```diff
@@ -0,0 +1,38 @@
+# Example config for jira_sync
```

Comment:
The legacy config could be removed completely.

Reply:
👍